.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY = "7b21cfd53f18535e1e40070e76eec614"
```

Comment:
never ever commit your .env file. Always add the .env into the .gitignore file first before doing anything else. Check with `git status` if your `.env` is included in the changes before committing anything. If it is included, you set up your gitignore wrongly